workflow/docker_util.py

```diff
@@ -0,0 +1,201 @@
+import docker
+import sys, re, os, shutil, tempfile, hashlib, json, base64, functools
+from pathlib import PurePath, Path
+
+from py2cytoscape import data
+from workflow import WorkflowRun
+from typing import Dict, List, Optional, Any, Type
+
+from workflow.runner import *
+
+
+CLIENTS = {
+    'TPS'       : 'ajshedivy/tps',

```

Comment:
I still may create a lab DockerHub organization.  If I do that before we merge, we can change these.